bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
+                    UNIT.MilliRAD,
+                    UNIT.MilliARCSEC,
+                    UNIT.REV,
+                ],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("Angle component of a phasor"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("PhasorAngle"),
+                SKOS.broader: QUDTQK.PlaneAngle,
+            },
+            "PhasorMagnitude": {
+                QUDT.applicableUnit: [
+                    UNIT.ARCMIN,
+                    UNIT.ARCSEC,
+                    UNIT.DEG,
+                    UNIT.GON,
+                    UNIT.GRAD,
+                    UNIT.MIL,
+                    UNIT.RAD,
+                    UNIT.MicroRAD,
+                    UNIT.MilliRAD,
+                    UNIT.MilliARCSEC,
+                    UNIT.REV,
+                ],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("Magnitude component of a phasor"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("PhasorMagnitude"),
+            },
+        },
     },
-    "Grains": {},
     "Power": {
-        "subclasses": {
+        OWL.sameAs: QUDTQK["Power"],
+        "subconcepts": {
             "Electric_Power": {
-                "subclasses": {
-                    "Apparent_Power": {},
-                    "Active_Power": {OWL.equivalentClass: BRICK["Real_Power"]},
-                    "Real_Power": {},
-                    "Reactive_Power": {},
-                    "Complex_Power": {},
+                OWL.sameAs: QUDTQK["ElectricPower"],
+                "subconcepts": {
+                    "Apparent_Power": {OWL.sameAs: QUDTQK["ApparentPower"]},
+                    "Active_Power": {
+                        OWL.sameAs: [QUDTQK["ActivePower"], BRICK["Real_Power"]],
+                    },
+                    "Real_Power": {
+                        OWL.sameAs: [QUDTQK["ActivePower"], BRICK["Active_Power"]],
+                    },
+                    "Reactive_Power": {OWL.sameAs: QUDTQK["ReactivePower"]},
+                    "Complex_Power": {OWL.sameAs: QUDTQK["ComplexPower"]},
                 },
             },
             "Peak_Power": {
+                SKOS.broader: QUDTQK.Power,
+                QUDT.applicableUnit: [UNIT.KiloW, UNIT.MegaW, UNIT.MilliW, UNIT.W],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L2I0M1H0T-3D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("PeakPower"),
                 SKOS.definition: Literal(
                     "Tracks the highest (peak) observed power in some interval"
                 ),
             },
-            "Thermal_Power": {},
+            "Thermal_Power": {
+                QUDT.applicableUnit: [UNIT.MilliW, UNIT.W, UNIT.KiloW, UNIT.MegaW],
```

Comment:
MBTU per hour is also common, unfortunately.

Reply:
Why "unfortunately"? QUDT has support for BTU; adding MBTU could be done here or in QUDT

Reply:
Yes, please add MBTU-per-hour here or in QUDT.